### DISCUSSION TITLE
Amazing work; non-recurring sponsorship?

### DISCUSSION THREAD

**Discussion by wbpaley** [dyn4j/dyn4j#232] [General]
This is a generous contribution to the community--and very nicely structured code; thank you!

Good to see you're still active on it.

I'd really like to contribute but am philosophically opposed to repeating sponsorship; is there a way I can give one (or more) one-time contributions?

**Comment by wnbittle** [comment]
Thank you very much for the feedback!  I work on it because I find enjoyment doing so and the sponsorship is just another motivation.  If you have an opposition to the _recurring_ sponsorship, there is an option for a one-time donation.  That said, I'd be just as happy if you gave that to a local charity near you.

**Reply by wbpaley** [reply]
Replied to the email, but don't see it here--maybe it didn't go through?

Anyway, I also added a "show & tell" discussion here in case anyone's interested in a non-gaming application.

---emailed answer---

Oops--didn't see that switch.

Yes, it's clear from how carefully written and documented the codebase is that this is a labor of love. (My own work is, too.)

I was hoping to donate more than $20. Don't be too humble! Your work has given me a huge amount of satisfaction and I just want to reflect back a bit of the value I've received.

Is there a way to tell GitHub to provide a typed-in amount? Or better: find me (Google "wbpaley") and email me an address I can PayPal to. (I try to limit the number of organizations that have my credit card info...)

With Great Appreciation,
Brad
